### PR TITLE
Make sure --img argument is a file accessible for reading and writing before preloading.

### PR DESCRIPTION
**Pull request by zvin**
Change-Type: patch

### CODE REVIEW

**Comment by zvin** [comment]
Connects to #51

File: bin/resin-preload [hunk 1]
```diff
@@ -114,6 +114,24 @@ if( !(options.appId && options.image && (options.apiToken || options.apiKey)) )
   process.exit(1)
 }
 
+const fs = require('fs')
+
+function isReadableAndWritable(path) {
+  try {
+    fs.accessSync(options.image, fs.constants.R_OK | fs.constants.W_OK)
```
Comment (jhermsmeier) [inline bin/resin-preload]: Looks like `options.image` should be `path` here

Comment (zvin) [inline bin/resin-preload]: good catch, fixed

**Comment by jhermsmeier** [comment]
@zvin don't forget to bump the version & publish ;)

**Comment by zvin** [comment]
@jhermsmeier done: https://www.npmjs.com/package/resin-preload

**Pull request merged**